iosApp/iosApp/ViewModels/ErrorBannerViewModel.swift

```diff
@@ -11,30 +11,39 @@ import shared
 
 class ErrorBannerViewModel: ObservableObject {
     let errorRepository: IErrorBannerStateRepository
+    let settingsRepository: ISettingsRepository
 
     @Published
     private(set) var errorState: ErrorBannerState? = nil
 
     @Published
     var loadingWhenPredictionsStale: Bool
 
+    @Published
+    var showDebugMessages: Bool = false
```

Comment:
I was playing around with different ideas of how to expose whether the debug setting is on or not - whether through some centralized state (ViewModel prop drilled through or EnvironmentObject). I opted for just adding it where it is needed for simplicity for this one piece, but may revisit as I add more